git-checkout-interactive

```diff
@@ -12,12 +12,14 @@ if [ ! -d ".git" ]; then
 fi
 
 # Fetch all recent branches sorted by recency of commit
-branches=$(git for-each-ref --count=30 --sort=-committerdate refs/heads/ --format='%(refname:short)')
+branches=$(git for-each-ref --count=30 --sort=-committerdate refs/heads/ refs/remotes/ --format='%(refname:short)' \
+  | sed -e s'/^origin\///' \
```

Comment:
What is sed doing here?

Reply:
Removes the prefix of `origin/` from the remote branch name so that the checkout works correctly. I guess `origin` might not actually be the name of the remote for everyone, so that may not work reliably. 

Goal was to be able to easily checkout remote branches so, for ex, if you have a remote called `origin` with a branch called `foo`, it's ordinarily returned as `origin/foo` but `git checkout origin/foo` doesn't work as expected, while simply running `git checkout foo` automatically creates a new local branch called `foo` from `origin/foo`

Reply:
Feel free to close the PR; I patched it and it's working for me locally, I just opened the PR out of habit. 

Thanks for the great script!

Reply:
Ok it makes sense.
Can you update the script so that the branch list generated distinguishes between local and origin branches? And when user wants to check out a branch on origin they must be informed about the fact that new local branch is being created

It could look something like this
```
> gcoi

1. development
2. some_trial
3. master
4. Create origin/foo
5. Create origin/staging
```
So when user selects say 4, we execute `git checkout foo`

Let me know if you want to suggest something else.

But good work although. Cheers